lib/resty/lock.lua

```diff
@@ -107,6 +108,7 @@ function _M.new(_, dict_name, opts)
     local self = {
         cdata = cdata,
         dict = dict,
+        dict_add = safe_add and dict.safe_add or dict.add,
```

Comment:
Code readability is not good enough.
how about this:
local dict_add = dict.add
if safe_add then
    dict_add = dict.safe_add
end